pipeline/acquisition.py

```diff
@@ -0,0 +1,362 @@
+'''
+Schema of aquisition information.
+'''
+import re
+import os
+from datetime import datetime
+import numpy as np
+import scipy.io as sio
+import datajoint as dj
+import h5py as h5
+
+from . import reference, subject, behavior, helper_functions
+
+schema = dj.schema(dj.config.get('database.prefix', '') + 'gi2017_acquisition')
+
+@schema
+class ExperimentType(dj.Lookup):
+    definition = """
+    experiment_type: varchar(64)
+    """
+    contents = [
+        ['behavior'], ['extracelluar'], ['photostim']
+    ]
+
+@schema 
+class ActionLocation(dj.Manual): 
+    definition = """ # Information relating the location of any experimental task (e.g. recording (extra/intra cellular), stimulation (photo or current) )
+    -> reference.BrainLocation
+    -> reference.CoordinateReference
+    coordinate_ap: float    # in mm, anterior positive, posterior negative 
```

Comment:
Do not use floats in the primary key.   Use decimal instead. This is because primary key attribute are often subject to equality comparisons.  Since SQL converts to decimal representation, floats often fail equality comparisons.